solidity/dashboard/src/components/resources/Navigation.jsx

```diff
@@ -7,43 +7,33 @@ const Navigation = () => {
     <nav className="tile resources-nav">
       <h5 className="text-grey-50">Contents</h5>
       <ul>
-        <ResourcesLink
-          to={{ pathname: "/resources", hash: "#documentation" }}
-          label="Documentation"
-        />
-        <ResourcesLink
-          to={{ pathname: "/resources", hash: "#quick-terminology" }}
-          label="Quick Terminology"
-        />
-        <ResourcesLink
-          to={{ pathname: "/resources", hash: "#diagram" }}
-          label="Delegation Diagram"
-          offset={-800}
-        />
+        <ResourcesLink to="documentation">Documentation</ResourcesLink>
+        <ResourcesLink to="diagram">Diagram</ResourcesLink>
+        <ResourcesLink to="quick-terminology">Quick Terminology</ResourcesLink>
       </ul>
     </nav>
   )
 }
 
-const ResourcesLink = ({ label, to, offset = -100 }) => {
+const ResourcesLink = ({ to, children, offset = -100 }) => {
   const { hash } = useLocation()
   const history = useHistory()
 
   return (
     <li>
       <Link
-        className={`text-small${
-          hash === to.hash ? " active" : ""
-        } cursor-pointer`}
+        className={`text-small${hash === to.hash ? " active" : ""}`}
         activeClass="active"
-        to={to.hash.slice(1)}
+        to={to}
         spy={true}
         smooth={true}
         offset={offset}
         duration={500}
-        onSetActive={() => history.replace(to)}
+        onSetActive={() =>
+          history.replace({ pathname: "/resources", hash: to })
```

Comment:
I guess the one downside is now this file has to know that resources are dropped at `/resources`... I think I'll have some more thoughts around this bundled into the suggestion regarding [sublinks in 2101](https://github.com/keep-network/keep-core/pull/2101#discussion_r500639319).

Reply:
Actually, it looks like we might should be able to use `let { path } = useRouteMatch();` to look up the parent route, maybe?